engine/chaos/manifests/network-emulation-dataflow.yaml

```diff
@@ -0,0 +1,113 @@
+---
+# A Network Loss action causes network packets to drop randomly
+apiVersion: chaos-mesh.org/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-loss-dataflow
+  labels:
+    app: network-loss-dataflow
+spec:
+  action: loss
+  mode: one
+  selector:
+    pods:
+      default: # default namespace
+        - server-master-0
+        - server-master-1
+        - server-master-2
+        - executor-0
+        - executor-1
+        - executor-2
+        - executor-3
+  loss:
+    loss: "25"
+    correlation: "25"
+  duration: "30s"
+  scheduler:
+    cron: "2-59/5 * * * *" # At every 5th minute from 2 through 59, (2, 7, 12, ...)
+
+
+# A Network Delay action causes delays in message sending
+---
+apiVersion: chaos-mesh.org/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-delay-dataflow
+  labels:
+    app: network-delay-dataflow
+spec:
+  action: delay
+  mode: one
+  selector:
+    pods:
+      default: # default namespace
+        - server-master-0
+        - server-master-1
+        - server-master-2
+        - executor-0
+        - executor-1
+        - executor-2
+        - executor-3
+  delay:
+    latency: "90ms"
```

Comment:
Should a larger delay be set here?

Reply:
Changed to 150ms, the max latency will be 300ms.
Given some experiments, 100ms latency will cause gPRC throughput decrease 25%, 200ms at least for 50%.